src/3rdparty/kissfft/test/testkiss.py

```
@@ -4,158 +4,140 @@
 #
 # SPDX-License-Identifier: BSD-3-Clause
 #  See COPYING file for more information.
-from __future__ import division,print_function
+from __future__ import absolute_import, division, print_function
```

Ruff (0.11.9)

Unnecessary `__future__` imports `absolute_import`, `division`, `print_function` for target Python version

Remove unnecessary `__future__` import

(UP010)